app/models/work.rb

```diff
@@ -120,20 +120,30 @@ def publish_test_doi?
 
     private
 
-      def works_by_user_state(user, state)
-        # The user's own works by state (if any)
-        works = Work.where(created_by_user_id: user, state: state).to_a
+      def search_terms_where_clause(search_terms)
+        if search_terms.nil?
+          Work.where("true")
+        else
+          Work.where("CAST(metadata AS VARCHAR) ilike :search_terms", search_terms: "%" + search_terms.strip + "%")
```

Comment:
```suggestion
          Work.where("CAST(metadata AS VARCHAR) ILIKE :search_terms", search_terms: "%" + search_terms.strip + "%")
```

Capitalize keywords.